src/main/kotlin/org/jitsi/jibri/api/http/internal/InternalHttpApi.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.jitsi.jibri.api.http.internal
+
+import org.jitsi.jibri.util.extensions.schedule
+import java.util.concurrent.Executors
+import java.util.concurrent.ScheduledExecutorService
+import java.util.logging.Logger
+import javax.ws.rs.POST
+import javax.ws.rs.Path
+import javax.ws.rs.core.Response
+
+@Path("/jibri/api/internal/v1.0")
+class InternalHttpApi(
+    private val executor: ScheduledExecutorService = Executors.newSingleThreadScheduledExecutor(),
```

Comment:
I see we have an easy way to have the threads named (used in XmppApi.kt). Is it intentionally not used here?

Reply:
i added this midway through and meant to go back and clean them all up to use it but forgot looks like.  thanks.